### PR TITLE
Fix the `make jobs` target

### PR DESCRIPTION
**Pull request by alanconway**
Signed-off-by: Alan Conway <aconway@redhat.com>

### CODE REVIEW

**Comment by alanconway** [comment]
The previous commands were broken (${CURDIR} undefined, missing images)
The new commands worked for me when setting up the knative-eventing-contrib 0.9.0 release CI but I'm not certain if they're compete for all cases.

File: Makefile [hunk 1]
```diff
@@ -44,9 +44,7 @@ services:
 # these are useful for devs
 jobs:
 	docker pull registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest
-	docker run -it -v "${CURDIR}/ci-operator:/ci-operator" registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest --from-dir /ci-operator/config --to-dir /ci-operator/jobs
-	docker pull registry.svc.ci.openshift.org/ci/determinize-prow-jobs:latest
-	docker run -it -v "${CURDIR}/ci-operator/jobs:/ci-operator/jobs" registry.svc.ci.openshift.org/ci/determinize-prow-jobs:latest --prow-jobs-dir /ci-operator/jobs
+	docker run -it -v "${PWD}/ci-operator:/ci-operator:z" registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest --from-dir /ci-operator/config --to-dir /ci-operator/jobs
```
Comment (petr-muller) [inline Makefile]: This does not look correct. `CURDIR` should work in `make`, it's preferrable to `PWD`. The second command is also supposed to work.

**Comment by petr-muller** [comment]
/hold

Pretty sure we should not do this change.

What exactly was "broken"? How did you run the make target? What was the actual error output?

**Comment by droslean** [comment]
@alanconway Please add more context in the description of this PR, to explain why we need this change?

**Comment by alanconway** [comment]
Closing, misconfiguration on my part I think. `make jobs` works for me. 
It does generate a lot of WARN messages which is not ideal but this PR is bogus.

**Pull request closed without merging**